packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -276,8 +276,16 @@ function build(syscall, _state, makeRoot, forVatID) {
     if (result) {
       lsdebug(` ls.deliver attaching then ->${result}`);
       insistVatType('promise', result);
+
+      // We return the results of the resolve/reject, rather
+      // than rejecting the dispatch with whatever the method
+      // did.
+      // It is up to the caller to handle or fail to handle the
+      // rejection.  Failing to handle should trigger a platform
+      // log, rather than via our parent doProcess call.
+
       // eslint-disable-next-line no-use-before-define
-      p.then(thenResolve(result), thenReject(result));
+      return p.then(thenResolve(result), thenReject(result));
```

Comment:
Does this belong in a different PR?

Reply:
Yup!

Reply:
that's the change that makes the kernel (well, liveslots) stop announcing vat promise rejections, right?

I guess if we're generally ok with teaching developers to use the new logging tools to hear about these, then I'm ok with this landing now. I opened #726 to discuss it, but I think we didn't raise any objections when talking about it this afternoon.

Reply:
Already separated out into #732